README.md

```diff
@@ -213,72 +150,4 @@ npm run build
 
 ## Testing
 
-### Installation
-
-CasperJS, PhantomJS, & BackstopJS all need to be installed globally:
-
-```sh
-npm install -g phantomjs casperjs backstopjs
-```
-
-### Run tests
-
-The project uses the visual regression testing system [BackstopJS](https://garris.github.io/BackstopJS/) to compare screenshots of each object before & after changes are made. For existing objects, if there is no visual difference between the reference images and the newly-generated images, your changes are good. For new objects, create new reference images (see below) and commit them to the repo.
-
-#### Existing objects
-
-If you haven’t added a new object to the `backstop.json` file, you can simply run the tests against the already-existing reference images:
-
-Start the styleguide server and leave it running:
-
-```sh
-npm run styleguide
-```
-
-To test your changes against the known-good reference images run the tests:
-
-```sh
-npm run test
-```
-
-Results from the comparison will be displayed in your browser in an automatically-opened window.
-
-### Before you create your PR
-
-Start up the styleguide (see the section below on testing for more information on this):
-
-```sh
-npm run styleguide
-```
-
-…then run the checks:
-
-```sh
-npm run checks
-```
-
-If you’ve made changes to the Sass source, also update the stats:
-
-```sh
-npm run css:stats
-```
-
-Once all tests pass, you’re good to go.
-
-#### New objects
-
-If you add new object scenarios to the `backstop.json`, backstop will complain if you try to run these tests without creating reference images for them.
-
-Start the styleguide server:
-
-```sh
-npm run styleguide
-```
-
-Then recreate the reference images:
-
-```sh
-npm run test:build
-```
-
-Check the new reference images match what you expect, then commit them and you’re done.
+Previously this project used a custom-made visual regression testing system. The dependencies used have sadly become abandoned in the years since, so they have been removed (encouraged by the Github security auditing feature that went live).
```

Comment:
Maybe add a reference to the issue that adds it back, in case somebody wants to track the progress?

Reply:
👍